ArduPlane/Parameters.cpp

```diff
@@ -644,14 +644,13 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     ASCALAR(min_gndspeed_cm,      "MIN_GNDSPD_CM",  MIN_GNDSPEED_CM),
 
-    // @Param: TRIM_PITCH_CD
+    // @Param: PITCH_TRIM
     // @DisplayName: Pitch angle offset
-    // @Description: Offset applied to AHRS pitch used for in-flight pitch trimming. Correct ground leveling is better than changing this parameter.
-    // @Units: cdeg
-    // @Range: -4500 4500
-    // @Increment: 10
-    // @User: Advanced
-    GSCALAR(pitch_trim_cd,        "TRIM_PITCH_CD",  0),
+    // @Description: Offset used for in-flight pitch trimming for level flight. Correct ground leveling is an alternative to changing this parameter.
+    // @Units: deg
+    // @Range: -45 45
+    // @User: Standard
+    ASCALAR(pitch_trim,        "PITCH_TRIM",  0.0f),
```

Comment:
I would greatly prefer TRIM_PITCH_DEG so users looking for the old parameter can find the new one easily

Reply:
I thought part of the point of getting rid of the "CD" was that the units are already in the metadata. Users can easily find the parameter by typing "TRIM" or "PITCH" in the search box.

Reply:
Oh and I just noticed this comment in the related issue - maybe it would be better to use PTCH because it's the most consistently used tag for "PITCH" which often needs to be abbreviated. https://github.com/ArduPilot/ardupilot/issues/23580#issuecomment-1519206557

Reply:
I agree with Tim, I don't think we need to add deg for si units. Adding deg also sets up a pattern we may not be able to follow as deg is one character longer than cd.

We could add a magnitude arming check, anything above say 10 we could assume the user copied the cd value.

Reply:
that shouldn't stop us adding _DEG when we can. It will save users a lot of pain and most params can get the _DEG

Reply:
There are already at least 13 PITCH parameters that are in degrees, but do NOT have "DEG" in the name, and none that do. Not to mention all the YAW and ROLL parameters that are in degrees with no "DEG" at the end.

IMO it's inconsistency that is most painful to users, and again - it's in the metadata, which is displayed in both Mission Planner and QGroundControl.

Reply:
Just noticed this one: Q_TRIM_PITCH: Quadplane AHRS trim pitch (in degrees, no _DEG) so TRIM_PITCH would be consistent with this. without it we will have

- TRIM_PITCH_DEG (in degrees)
- Q_TRIM_PITCH (in degrees)

Reply:
Ok - I get it _DEG is necessary. So I've reworked this now.